doc/input.md

```diff
@@ -102,13 +102,13 @@ class MyGame extends Game with TapDetector {
   // Other methods omitted
```

Comment:
Maybe a section in this file explaining the event info, like what `position.game/widget/global` refers to?